composer/trainer/mosaic_fsdp.py

```diff
@@ -69,16 +69,6 @@ def patch_pytorch():
         from torch.distributed.fsdp import _runtime_utils
         _runtime_utils._validate_and_get_hybrid_shard_state = lambda *args, **kwargs: None
 
-        # Monkeypatch dtensor support
```

Comment:
Is this already in torch 2.2?

Reply:
It's buggy so we wont support in torch 2.2